src/Shared/EmptyState.js

```diff
@@ -170,19 +198,20 @@ const styles = StyleSheet.create({
     flex: 0.05,
   },
   image: {
-    width: null,
-    height: 300,
+    width: '100%',
+    height: '100%',
     flex: 1,
+    resizeMode: 'contain',
```

Comment:
should you just change this in the image declaration instead of having "cover" up there and "contain" down here?